commands/dataset/versions/create/mod.ts

```diff
@@ -0,0 +1,98 @@
+import { datasets, datasetVersions } from "../../../../api/dataset.ts";
+import { fields } from "../../../../flags.ts";
+import { asserts } from "../../../../lib/asserts.ts";
+import { dataTable } from "../../../../lib/data-table.ts";
+import { loading } from "../../../../lib/loading.ts";
+import { pickJson } from "../../../../lib/pick-json.ts";
+import { input } from "../../../../prompts/input.ts";
+import { select } from "../../../../prompts/select.ts";
+import { z } from "../../../../zcli.ts";
+import { args } from "../../../../zcli.ts";
+import { command, flag, flags } from "../../../../zcli.ts";
+import { defaultFields } from "../mod.ts";
+
+/**
+ * This variable is automatically generated by `zcli add`. Do not remove this
+ * or change its name unless you're no longer using `zcli add`.
+ */
+const subCommands: ReturnType<typeof command>[] = [];
+
+export const create = command("create", {
+  short: "Create a dataset version",
+  long: `
+    Create a dataset version.
```

Comment:
Should you add "This command will prompt you for a dataset if you don't provide one."